apps/tsforjs/src/components/Accordion.tsx

```diff
@@ -0,0 +1,107 @@
+import React, {useState} from 'react'
+import Image from 'next/image'
+
+export default function Accordion() {
```

Comment:
Something to consider when reaching for these types of patters is checking Radix. They have a lot of core components and start from an accessible baseline that can make better experience css with less effort

I didn't dig in deep to see if it's a fit here, but saw "accordion" and wanted to share https://www.radix-ui.com/docs/primitives/components/accordion

Reply:
@joelhooks, great suggestion, so much cleaner and easy! I implemented the accordion using Radix. 

https://github.com/skillrecordings/products/pull/399

In that PR you can see the new accordion, and I'm also deleting all the unused code.